2011-10-21  Carlos O'Donell  <carlos@systemhalted.org>

	* ports/sysdeps/unix/sysv/linux/hppa/sys/procfs.h: Do 
	not include signal.h and sys/ucontext.h. 

---
 ports/sysdeps/unix/sysv/linux/hppa/sys/procfs.h |    2 --
 1 file changed, 2 deletions(-)

--- a/ports/sysdeps/unix/sysv/linux/hppa/sys/procfs.h
+++ b/ports/sysdeps/unix/sysv/linux/hppa/sys/procfs.h
@@ -29,10 +29,8 @@
    GDB unless you know what you are doing.  */
 
 #include <features.h>
-#include <signal.h>
 #include <sys/time.h>
 #include <sys/types.h>
-#include <sys/ucontext.h>
 #include <sys/user.h>
 
 __BEGIN_DECLS
